website/content/cloud-native-sustainability-week/_index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: CNCF Cloud Native Sustainability Week
+linkTitle: Cloud Native Sustainability Week
+toc_hide: true
+exclude_search: true
+menu:
+  main:
+    weight: 30
+description: Cloud Native Sustainability Week is a global event where CNCF community organize local meetups around the topic of cloud native sustainability. Cloud Native Sustainability Week will take place during the second week of October 2023.
+slug: cloud-native-sustainability-week
+---
+
+The CNCF Global Week of Cloud Native Sustainability is an event organized by the [Cloud Native Computing Foundation (CNCF)](http://cncf.io) community to address the emerging topic of environmental sustainability in the cloud native industry and open source space. The event aims to engage with the community and IT industry as a whole, get a better understanding of the current environmental sustainability landscape, and promote collaboration and knowledge sharing on the topic.
+
+During the **second week of October 2023 (W41)**, the [CNCF community groups around the globe](https://community.cncf.io/chapters/) will organize meetups in their cities with a focus on cloud native sustainability. These meetups can be organized by anyone interested in supporting the effort. Additionally, the [CNCF Environmental Sustainability Technical Advisory Group (TAG ENV)](http://github.com/cncf/tag-env-sustainability), will host a virtual meetup using the [bevy](http://bevy.com) platform.
+These meetups can be organized by anyone interested in supporting the effort. If you would like to contribute, check out the [Contact and Resources](#contact-and-resources) section below.
+
+There will be two types of events organized during the Cloud Native Sustainability Week:
+1. **Local in-person events**: Meetups organized worldwide during the second week of October 2023 by [CNCF community groups](https://community.cncf.io/chapters/) about cloud native sustainability.
+2. **Virtual Event**: A virtual meetup organized by the CNCF TAG ENV.
+
+Please take a look at this [document](https://docs.google.com/document/d/1s28lqqc3IBAMw4T7n13zfyAuUEEc3I4_xhPxCLA4_dk/edit?usp=sharing) for a more detailed overview.
+
+## Meetup Locations
+
+*Note that this list is changing, we will publish a complete overview a few weeks for the event! Take a look at this [document](https://docs.google.com/document/d/1s28lqqc3IBAMw4T7n13zfyAuUEEc3I4_xhPxCLA4_dk/edit#heading=h.19phjl5j6fdw) to check the latest status, and persons to contact.*
+
+| **Nr** | **Name** | **Location** | **Date** | **Event Link** |
+|---|---|---|---|---|
```

Comment:
Can we group these by continent, maybe?  I'd probably do location first, date, event, then name (this is the relevance I do).

Reply:
good idea